src/main/java/uk/gov/hmcts/probate/transformer/solicitorexecutors/SolicitorApplicationCompletionTransformer.java

```diff
@@ -42,4 +44,10 @@ public void mapSolicitorExecutorFieldsOnAppDetailsComplete(CaseData caseData) {
         formatFields(caseData);
         mapSolicitorExecutorFieldsToLegalStatementExecutorFields(caseData);
     }
+
+    public void eraseCodicilAddedDateIfWillHasCodicilIsNo(CaseData caseData) {
```

Comment:
bit of a strange name for this method
eraseCodicilAddedDateIfWillHasNoCodicils
maybe?